packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -136,6 +140,13 @@ export class Viewer {
    */
   @Prop() public cameraControls = true;
 
+  /**
+   * Enables or disables the default keyboard shortcut interactions provided by
+   * the viewer. Enabled by default, requires `cameraControls` being enabled.
+   *
+   */
+  @Prop() public keyboardControls = true;
```

Comment:
Wondering if we'll want a customizable payload here to specify which ones you might want? Just a thought. Or do we want just a kill switch to shut them all off?

Reply:
Yeah, that's a good question. These changes follow the pattern we have for camera controls, but maybe we'd want more control over these. Trying to think through a way to avoid having to add another "Config" type prop, curious what you would think about something like:
```typescript
type KeyboardControls = 'default' | 'custom' | 'none';
```
and then exporting the individual `KeyInteraction`s, so in the case that you want some interactions but not others, you could do something like:
```typescript
<vertex-viewer keyboard-controls="custom" />

viewer.registerTapKeyInteraction(
  new FlyToPartKeyInteraction(),
  new ShiftSelectKeyInteraction(),
  // etc
)
```
might require a bit of refactoring, but that would allow customization of the interactions without passing an object description of key interactions, and without having to register the key state interaction handler

Reply:
This is likely something we could defer and prioritize when we add additional controls. Probably nothing needed now for this.